modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java

```diff
@@ -816,6 +819,8 @@ private ObjectField _addObjectField(
 		ObjectDefinition objectDefinition =
 			_objectDefinitionPersistence.findByPrimaryKey(objectDefinitionId);
 
+		_validateBundleNamespace(objectDefinition.isModifiable(), name, system);
```

Comment:
I believe we don't need because the `addOrUpdateSystemObjectField()` calls the `_addObjectField()` method when is adding a system object field and when updating the system object field, only the label can be updated.